### ISSUE TITLE
Compliance with City and WCAG standards

### ISSUE THREAD

**Issue by JALogsdon** [JALogsdon/cosd-arcgis-thumbnail-builder#1]
Design and accessibility should be reviewed and updated to comply with the [City Visual Guide](https://www.sandiego.gov/communications/design) and [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/).
Update colors, fonts, and improve other accessibility elements.
